Resources/doc/1-configuration-reference.md

```diff
@@ -54,44 +70,22 @@ __Supported algorithms for OpenSSL:__
 __Supported algorithms for phpseclib:__
 - RS256, RS384, RS512 (RSA)
 
-Security reference
--------------------
-
-### Simplest configuration
-
-``` yaml
-# app/config/security.yml
-# ...
-firewalls:
-    # ...
-    api:
-        # ...
-        lexik_jwt: ~ # check token in Authorization Header, with a value prefix of 'Bearer'
-```
+Security configuration
+-----------------------
 
 ### Full default configuration
 
-``` yaml
+```yaml
 # app/config/security.yml
 # ...
 firewalls:
     # ...
     api:
         # ...
-        # advanced configuration
-        lexik_jwt:
-            authorization_header: # check token in Authorization Header
-                enabled: true
-                prefix:  Bearer
-                name:    Authorization
-            cookie:               # check token in a cookie
-                enabled: false
-                name:    BEARER
-            query_parameter:      # check token in query string parameter
-                enabled: false
-                name:    bearer
-            throw_exceptions:        false     # When an authentication failure occurs, return a 401 response immediately
-            create_entry_point:      true      # When no authentication details are provided, create a default entry point that returns a 401 response
-            authentication_provider: lexik_jwt_authentication.security.authentication.provider
-            authentication_listener: lexik_jwt_authentication.security.authentication.listener
+        guard:
+            authenticators: 
+                - lexik_jwt_authentication.jwt_token_authenticator
 ```
+
+For more details about the `lexik_jwt_authentication.jwt_token_authenticator` service and how to
+customize it, see ["Extending the Guard JWTTokenAuthenticator"](6-extending-jwt-authenticator.md)
```

Comment:
Not so big at the moment but [here](https://github.com/lexik/LexikJWTAuthenticationBundle/blob/guard_jwt_authenticator/Resources/doc/6-extending-jwt-authenticator.md) you can read it :)